.github/workflows/release.yml

```diff
@@ -0,0 +1,136 @@
+name: Release tests
```

Comment:
Hi @ana before I do a complete review, by looking at glance, one thing that caught my attention here in this file is the mix between target calls vs code here.

IMO it is better to concentrate whenever possible the code inside Makefile/setup.py and here just make a call for the target, as you did with `make rpm-release` and `make wheel`.

Reply:
@beraldoleal Please, don't review this PR! I have continued working and it has changed a lot, see https://github.com/ana/avocado-test
Replying to your question: that actions are only required for the release, it's better to have them only in the release action.

Reply:
Even for actions required only during the release, IMO does not justifies to have the code inside the .yml file.

I will wait for the final PR then.